src/ts/directives/virtual-folder/virtual-media-library.directive.ts

```diff
@@ -125,6 +130,15 @@ class Controller implements ng.IController, IVirtualMediaLibrary {
         };
     }
 
+    private initFolderCache = (folderService: Tree) => {
```

Comment:
Pour justement éviter l'erreur TypeError : https://github.com/opendigitaleducation/infra-front/blob/dev/src/ts/directives/folderTree.ts#L97

`TypeError: Cannot read properties of undefined (reading 'isEmpty')`
 au niveau de `cacheChildren`